tests/ovs-macros.at

```diff
@@ -226,14 +226,9 @@ m4_divert_pop([PREPARE_TESTS])
 
 OVS_START_SHELL_HELPERS
 ovs_cleanup() {
-    if test "$(echo asan.*)" != 'asan.*'; then
-        echo "Address Sanitizer reported errors in:" asan.*
-        cat asan.*
-        AT_FAIL_IF([:])
-    fi
-    if test "$(echo ubsan.*)" != 'ubsan.*'; then
-        echo "Undefined Behavior Sanitizer reported errors in:" ubsan.*
-        cat ubsan.*
+    if test "$(echo sanitizers.*)" != 'sanitizers.*'; then
```

Comment:
This is incorrect.  In atlocal.in we have:
```
# Add some default flags to make the tests run better under Address
# Sanitizer, if it was used for the build.
ASAN_OPTIONS=detect_leaks=1:abort_on_error=true:log_path=asan:$ASAN_OPTIONS
export ASAN_OPTIONS

# Add some default flags for UndefinedBehaviorSanitizer, if it was used
# for the build.
UBSAN_OPTIONS=print_stacktrace=1:halt_on_error=true:log_path=ubsan:$UBSAN_OPTIONS
export UBSAN_OPTIONS
```
So any sanitizer error reports will be in `asan.*` or in `ubsan.*` (depending on which log path is taken into account first by the sanitizer).

With this change test failures that are due to sanitizer reported bugs will not be detected by the testsuite and can potentially be hidden by the re-test (we used to skip re-test if asan/ubsan reported problems).

As a matter of fact it's what's happened in [your older clang run](https://github.com/ovn-org/ovn/runs/7218725495?check_suite_focus=true#step:13:7002):
```

=================================================================
==1630936==ERROR: LeakSanitizer: detected memory leaks

Direct leak of 64 byte(s) in 1 object(s) allocated from:
    #0 0x49a1c9 in realloc (/home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/utilities/ovn-nbctl+0x49a1c9)
    #1 0x758aa5 in xrealloc__ /home/runner/work/ovn/ovn/ovs/lib/util.c:147:9
    #2 0x758aa5 in xrealloc /home/runner/work/ovn/ovn/ovs/lib/util.c:179:12
    #3 0x758aa5 in x2nrealloc /home/runner/work/ovn/ovn/ovs/lib/util.c:239:12
    #4 0x751208 in svec_expand /home/runner/work/ovn/ovn/ovs/lib/svec.c:92:23
    #5 0x751208 in svec_terminate /home/runner/work/ovn/ovn/ovs/lib/svec.c:116:5
    #6 0x7031d1 in ovs_cmdl_env_parse_all /home/runner/work/ovn/ovn/ovs/lib/command-line.c:98:5
    #7 0x4c9df1 in ovn_dbctl_main /home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/../../utilities/ovn-dbctl.c:132:20
    #8 0x4d2033 in main /home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/../../utilities/ovn-nbctl.c:7588:12
    #9 0x7fd6e17dd082 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x24082)

Indirect leak of 10 byte(s) in 1 object(s) allocated from:
    #0 0x499ead in malloc (/home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/utilities/ovn-nbctl+0x499ead)
    #1 0x75891e in xmalloc__ /home/runner/work/ovn/ovn/ovs/lib/util.c:137:15
    #2 0x75891e in xmalloc /home/runner/work/ovn/ovn/ovs/lib/util.c:172:12
    #3 0x75891e in xmemdup0 /home/runner/work/ovn/ovn/ovs/lib/util.c:193:15
    #4 0x75891e in xstrdup /home/runner/work/ovn/ovn/ovs/lib/util.c:202:12
    #5 0x750f61 in svec_add /home/runner/work/ovn/ovn/ovs/lib/svec.c:71:27
    #6 0x70318b in ovs_cmdl_env_parse_all /home/runner/work/ovn/ovn/ovs/lib/command-line.c:91:5
    #7 0x4c9df1 in ovn_dbctl_main /home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/../../utilities/ovn-dbctl.c:132:20
    #8 0x4d2033 in main /home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/../../utilities/ovn-nbctl.c:7588:12
    #9 0x7fd6e17dd082 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x24082)

Indirect leak of 8 byte(s) in 2 object(s) allocated from:
    #0 0x499ead in malloc (/home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/utilities/ovn-nbctl+0x499ead)
    #1 0x75891e in xmalloc__ /home/runner/work/ovn/ovn/ovs/lib/util.c:137:15
    #2 0x75891e in xmalloc /home/runner/work/ovn/ovn/ovs/lib/util.c:172:12
    #3 0x75891e in xmemdup0 /home/runner/work/ovn/ovn/ovs/lib/util.c:193:15
    #4 0x75891e in xstrdup /home/runner/work/ovn/ovn/ovs/lib/util.c:202:12
    #5 0x750f61 in svec_add /home/runner/work/ovn/ovn/ovs/lib/svec.c:71:27
    #6 0x7031bb in ovs_cmdl_env_parse_all /home/runner/work/ovn/ovn/ovs/lib/command-line.c:96:9
    #7 0x4c9df1 in ovn_dbctl_main /home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/../../utilities/ovn-dbctl.c:132:20
    #8 0x4d2033 in main /home/runner/work/ovn/ovn/ovn-22.06.90/_build/sub/../../utilities/ovn-nbctl.c:7588:12
    #9 0x7fd6e17dd082 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x24082)

SUMMARY: AddressSanitizer: 82 byte(s) leaked in 4 allocation(s).
```
```